lib/tasks.ex

```diff
@@ -1,2 +1,14 @@
 defmodule Tasks do
+  def fizz_buzz(n) when n < 1 or n > 100 do
+    n
+  end
+
+  def fizz_buzz(n) do
+    case {rem(n, 3) == 0, rem(n, 5) == 0} do
+      {false, true} -> "Buzz"
+      {true, false} -> "Fizz"
+      {true, true} -> "FizzBuzz"
+      {false, false} -> n
+    end
```

Comment:
You can skip equality check and do something like this:

```elixir
   case {rem(n, 3), rem(n, 5), n} do
      {0, 0, _} -> "FizzBuzz"
      {0, _, _} -> "Fizz"
      {_, 0, _} -> "Buzz"
      {_, _, n} -> n
    end
```

Your solution is perfectly fine, probably even a bit more readable. I hope you see and understand how I changed the order of the clauses. Pattern matching is great stuff.